tests/Rector/Class_/AddUseAnnotationToHasFactoryTraitRector/Fixture/deep_namespace.php.inc

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Models\Tenant;
```

Comment:
Please use the correct namespace here, which would be `namespace RectorLaravel\Tests\Rector\Class_\AddUseAnnotationToHasFactoryTraitRector\Fixture;`.
To make the test work you would have to make the `use` annotation a bit different:
```
@use \Illuminate\Database\Eloquent\Factories\HasFactory<\RectorLaravel\Tests\Rector\Class_\AddUseAnnotationToHasFactoryTraitRector\Source\Tenant\UserFactory>
```
I know it does not look like the the real thing with `App\Models`, but wrong namespaces in tests have caused us issues in the past.